internal/datacoord/server.go

```diff
@@ -254,7 +254,30 @@ func (s *Server) Init() error {
 	var err error
 	s.stateCode.Store(commonpb.StateCode_Initializing)
 	s.factory.Init(&Params)
+	if err = s.initSession(); err != nil {
+		return err
+	}
+	if s.enableActiveStandBy {
+		s.activateFunc = func() {
+			log.Info("DataCoord switch from standby to active, activating")
+			if err := s.initDataCoord(); err != nil {
+				log.Warn("DataCoord init failed", zap.Error(err))
+				// TODO: panic if error occurred?
```

Comment:
I have another pr about active-standy. activeFunc is updated to `func() error` then we can throw this error

Reply:
good job, LGTM